sdk/purview/Azure.Analytics.Purview.DataMap/src/Azure.Analytics.Purview.DataMap.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the Azure.Analytics.Purview.DataMap client library for developing .NET applications with rich experience.</Description>
+    <AssemblyTitle>Azure SDK Code Generation Azure.Analytics.Purview.DataMap for Azure Data Plane</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
+    <PackageTags>Azure.Analytics.Purview.DataMap</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>$(NoWarn);AZC0012;AZC0030</NoWarn>
```

Comment:
What are we suppressing here?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/26702#issuecomment-1928651120
This is to suppress the issue mentioned here.

Examples:
AutoCompleteOptions.cs(13,26): error AZC0030: Model name 'AutoCompleteOptions' ends with 'Options'. Suggest to rename it to 'AutoCompleteConfig' or any other appropriate name.
Generated/Discovery.cs(19,26): error AZC0012: Single word class names are too generic and have high chance of collision with BCL types or types from other libraries